weave-js/src/assets/icons/icon-crown-pro.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="14" height="14" viewBox="0 0 14 14" fill="none">
```

Comment:
i also think that we normally want the svgs to be width 24 height 24 and viewbox 0 0 24 24, but maybe thats just not available

Reply:
theres a 24x24 one here

https://www.figma.com/file/01KWBdMZg5QM9SRS1pQq0z/Design-System----Robot-Styles?type=design&node-id=8245-7172&mode=design&t=KuG9RevV3OUG9Oon-4

Reply:
For consistency we usually ask the design team to export at 24x24 and share via their Google Drive folder. The addicon script in the devtools repo can then take care of things like updating the fill value for you.